Testing/SystemTests/tests/analysis/complexShapeAbsorptionsTest.py

```diff
@@ -0,0 +1,44 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
Minor quibble. Could we call this `Testing/SystemTests/tests/analysis/ComplexShapeAbsorptionsTest.py` with the capital `C`?